### ISSUE TITLE
Reproducing the unittest ouptut for the test_abs function

### ISSUE THREAD

**Issue by andy1122** [unifyai/ivy#347]
System configurations:

Windows 10 Home
Pycharm + docker

After setting up the docker, i was able to run this command on windows:
docker run --rm -v /c/users/andy/ivy:/ivy unifyai/ivy:latest python3 -m pytest ivy/ivy_tests/
(Not all test got passed, which is not even a requirement for the current task)

I have forked the ivy repo yesterday and set the python interpeter so that it can use the docker image for python envirnment. 

https://www.youtube.com/watch?v=HNV4eI0zp4M&list=PLwNuX3xB_tv96FaEUmTirlQLIsnUgvtTl&index=1&ab_channel=Ivy

I am trying to follow above video and reproduce the results of the test_abs function. I have set numpy as array_module by uncommenting this line:
import numpy as array_module
But the test failed and i recieved the below ouptut:

3da445a4a750:python3 -u /opt/.pycharm_helpers/pycharm/_jb_pytest_runner.py --target /ivy/ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py::test_abs
Testing started at 11:38 PM ...
Launching pytest with arguments /ivy/ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py::test_abs --no-header --no-summary -q in /ivy

============================= test session starts ==============================
collecting ... 
ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py:None (ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py)
ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py:12: in <module>
    from . import array_helpers as ah
ivy_tests/test_array_api/array_api_tests/array_helpers.py:10: in <module>
    from . import dtype_helpers as dh
ivy_tests/test_array_api/array_api_tests/dtype_helpers.py:135: in <module>
    if dtype_nbits[default_int] == 32:
E   KeyError: dtype('int64')

collected 0 items / 1 error
ERROR: not found: /ivy/ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py::test_abs
(no name '/ivy/ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py::test_abs' in any of [<Module test_operators_and_elementwise_functions.py>])

========================= 1 warning, 1 error in 0.45s ==========================

Process finished with exit code 4

 This seems to be very silly, but i am not able to figure out how to solve this.

**Comment by Ishticode** [comment]
Shouldn't it be
`import ivy as array_module`
`array_module.set_framework("numpy")`
rather than
`import numpy as array_module`

**Comment by andy1122** [comment]
Hi @Ishticode 
I have commented this two lines:
    #import numpy as array_module  
    #arrary_module = None

and appended this two lines and this time the error message is different
   import ivy as array_module
   array_module.set_framework("numpy")


Launching pytest with arguments /ivy/ivy_tests/test_array_api/array_api_tests/test_operators_and_elementwise_functions.py::test_abs --no-header --no-summary -q in /ivy

ImportError while loading conftest '/ivy/ivy_tests/test_array_api/conftest.py'.
ivy_tests/test_array_api/conftest.py:7: in <module>
    from array_api_tests import _array_module as xp
ivy_tests/test_array_api/array_api_tests/__init__.py:6: in <module>
    from ._array_module import mod as _xp
ivy_tests/test_array_api/array_api_tests/_array_module.py:7: in <module>
    import ivy as array_module
ivy/__init__.py:55: in <module>
    from . import stateful
ivy/stateful/__init__.py:1: in <module>
    from . import activations
ivy/stateful/activations.py:7: in <module>
    from ivy.stateful.module import Module
ivy/stateful/module.py:9: in <module>
    import ivy.functional.backends.numpy
ivy/functional/backends/numpy/__init__.py:56: in <module>
    from . import array_api
ivy/functional/backends/numpy/array_api/__init__.py:3: in <module>
    from . import creation_functions
ivy/functional/backends/numpy/array_api/creation_functions.py:8: in <module>
    from ivy.functional.backends.numpy.core.general import _to_dev
ivy/functional/backends/numpy/core/__init__.py:17: in <module>
    from . import reductions
ivy/functional/backends/numpy/core/reductions.py:7: in <module>
    import numpy.array_api as _npa
E   ModuleNotFoundError: No module named 'numpy.array_api'

Process finished with exit code 4

Empty suite

**Comment by Ishticode** [comment]
This should be easy to deal with. You need to pull your docker image again and delete the old one and reset the interpreter. It is because of version difference. The new docker image has the current latest version (1.22.2) of numpy which does have numpy.array_api.

**Comment by andy1122** [comment]
Yes, i checked the numpy version is 1.19.5 in current image of docker. Let me pull the docker image again and i will update you after testing.

**Comment by andy1122** [comment]
After pulling the latest image of ivy(4 GB) from docker. I was able to pass the tests for the for the test_abs function. Now i can start working on the actual function.

Well there was one more open issue i found, since i was having shortage of space in my c drive. As soon as i deleted the docker image(10 GB) from my system via power shell, i expect to see an increase of 10 GB space in my c drive but that didn't happen. I found that it's an open issue with the docker windows and there is still no concrete solution for this.
https://github.com/docker/for-win/issues/244

Anyways, its just the docker related issue. Just in case anyone can shed some light on this, else we can close this issue.

Regards
Andy
